### PR TITLE
Update info about Julia Solórzano

### PR DESCRIPTION
**Pull request by juliaelman**
## Description

Hi! I am updating a few quick items around my bio, links and references.

### CODE REVIEW

File: _posts/2023-08-30-julia-solorzano.md [hunk 1]
```diff
@@ -43,10 +43,12 @@ Julia's Talk is titled "Django the Dinosaur: A Story About A Web Framework."
 
 If you are interested in learning more about Julia, you can find her on the web at:
 
-[juliaelman.com](https://juliaelman.com/)
+[juliasolorzano.com](juliasolorzano.com/)
```
Comment (CuriousLearner) [inline _posts/2023-08-30-julia-solorzano.md:46]: This link seems to be redirecting to the older link, so might be worth keeping the old one?

**Review by CuriousLearner — COMMENTED** [review]
One suggestion, rest looks good 💯

File: _posts/2023-08-30-julia-solorzano.md [hunk 2]
```diff
@@ -15,16 +15,16 @@ I recently sat down to chat with Julia about her upcoming keynote. She is passio
 
 ### Here's a little more about Julia:
 
-Julia Solórzano is the Security Experience Supervisor for [Login.gov](https://login.gov/) and is based in Durham, NC. She leads security, user experience and engineering teams, and coaches people to help them reach their full potential to create amazing things. Outside of Login.gov, her most recent work includes the [Department of Veterans API](https://developer.va.gov/) program and the [U.S. Web Design System](https://designsystem.digital.gov/). She is also a professor at Georgetown University where she teaches technology design and development.
+Julia Solórzano is the Security Experience Supervisor based in Durham, NC. She leads security, user experience and engineering teams, and coaches people to help them reach their full potential to create amazing things. Outside of working on [Login.gov](https://login.gov/), her most recent work includes the [Department of Veterans API](https://developer.va.gov/) program and the [U.S. Web Design System](https://designsystem.digital.gov/). She is also a professor at Georgetown University where she teaches technology design and development.
```
Comment (KatherineMichel) [inline _posts/2023-08-30-julia-solorzano.md:18]: Better: "Julia Solórzano is a Security..."

File: _posts/2023-08-30-julia-solorzano.md [hunk 3]
```diff
@@ -15,16 +15,16 @@ I recently sat down to chat with Julia about her upcoming keynote. She is passio
...
 
 Earlier in her career, Julia published a book called [Lightweight Django](https://www.oreilly.com/library/view/lightweight-django/9781491946275/) with [O’Reilly Media](https://www.oreilly.com/). She has also worked with clients such as Mozilla, the Python Software Foundation, University of Chicago and Product(RED).
 
 
-Julia's Talk is titled "Django the Dinosaur: A Story About A Web Framework." 
+Julia's Talk is titled "Django the Dinosaur: A Story About A Web Framework."
```
Comment (KatherineMichel) [inline _posts/2023-08-30-julia-solorzano.md:23]: Should be "talk" instead of "Talk"?

File: _presenters/julia-solorzano.md [hunk 4]
```diff
@@ -10,14 +10,16 @@ twitter: null
 website: null
 ---
 
-Julia Solórzano is the Security Experience Supervisor for [Login.gov](https://login.gov/) and is based in Durham, NC. She leads security, user experience and engineering teams, and coaches people to help them reach their full potential to create amazing things. Outside of Login.gov, her most recent work includes the [Department of Veterans API](https://developer.va.gov/) program and the [U.S. Web Design System](https://designsystem.digital.gov/). She is also a professor at Georgetown University where she teaches technology design and development.
+Julia Solórzano is the Security Experience Supervisor is based in Durham, NC. She leads security, user experience and engineering teams, and coaches people to help them reach their full potential to create amazing things. Outside of Login.gov, her most recent work includes [Login.gov](https://login.gov),  [Department of Veterans API](https://developer.va.gov/) program and the [U.S. Web Design System](https://designsystem.digital.gov/). She is also an adjunct professor at Georgetown University where she teaches technology design and development.
```
Comment (KatherineMichel) [inline _presenters/julia-solorzano.md:13]: Same as above: "Julia Solórzano is a Security..."
Also: you've got an extra Login.gov reference.

File: _posts/2023-08-30-julia-solorzano.md (hunk 3 above)
Comment (drewbrew) [reply _posts/2023-08-30-julia-solorzano.md:23]: Yes

File: _posts/2023-08-30-julia-solorzano.md (hunk 1 above)
Comment (drewbrew) [reply _posts/2023-08-30-julia-solorzano.md:46]: No, keep the new link because that's her new surname (recent marriage)

**Comment by tim-schilling** [comment]
Hi @juliaelman, I'm going to implement the suggestions Kati made and merge this. I want to get the final version of bios and talks to our online platform this evening and this is the only pending change remaining. I'm sure we can revert them if needed.

**Comment by tim-schilling** [comment]
I didn't have permissions to edit your remote branch so the merge didn't close this PR.

Closing this as it was merged in 04eb5f13213b9512fa974d7d8ce90997804e13b3

**Pull request closed without merging**